### ISSUE TITLE
Mobile menu expand/collapse a11y

### ISSUE THREAD

**Issue by aviannamiller** [omeka/theme-berlin#34]
The mobile menu is lacking the proper markup to communicate its expand/collapse behavior. `aria-expanded` should be present, at minimum.

**Comment by kimisgold** [comment]
There's a fix ready to test.

**Comment by aviannamiller** [comment]
Great, thanks!
